### PR TITLE
Vehicle Transfer

### PR DESCRIPTION
**Pull request by anowak2692**
Functionality and tests for being able to transfer a vehicle between users

### CODE REVIEW

File: src/main/java/com/revature/maadcars/controllers/VehicleController.java [hunk 1]
```diff
@@ -128,4 +139,27 @@ public VehicleController(VehicleService vehicleService){
         logger.trace("Successfully deleted vehicle with id: " + vehicle_id);
         return ResponseEntity.ok().build();
     }
+
+    /**
+     * Maps "PUT vehicles/transfer/{vehicle_id}/to/{new_user_id}"
```
Comment (mthuynguyen) [inline src/main/java/com/revature/maadcars/controllers/VehicleController.java:144]: Document what this method does with this mapping.

**Review by mthuynguyen — CHANGES_REQUESTED** [review]
Unit tests OK. Needs more documentation in 1 spot.

File: src/main/java/com/revature/maadcars/controllers/VehicleController.java (hunk 1 above)
Comment (anowak2692) [reply src/main/java/com/revature/maadcars/controllers/VehicleController.java]: Updated, should be there now.

**Review by mthuynguyen — APPROVED** [review]
lgtm

**Pull request merged**